src/components/SocialBtns/fb_sdk.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+module.exports = {
+  init: function() {
+    window.fbAsyncInit = function() {
+      FB.init({
+        appId      : '339880856413925',
```

Comment:
I'm not familiar with implementing sharing so this is new territory for me.  A couple thoughts/questions:
1.) Is this `appId` secret / do we want it committed to version control?
2.) We're using ES2015 imports in the frontend code, could you switch this over?
3.) Our file-naming convention is to camel case everything, so this would be something like `fbSdk.js` or `fbSDK.js`